.github/actions/sw-java-agent/action.yml

```diff
@@ -30,5 +30,5 @@ runs:
     - name: Download binary distribution
       shell: bash
       run: |
-        curl https://dlcdn.apache.org/skywalking/java-agent/${{ inputs.version }}/apache-skywalking-java-agent-${{ inputs.version }}.tar.gz -o apache-skywalking-java-agent.tar.gz
+        curl https://dlcdn.apache.org/skywalking/java-agent/${{ inputs.version }}/apache-skywalking-java-agent-${{ inputs.version }}.tgz -o apache-skywalking-java-agent.tar.gz
```

Comment:
Need to download from here, https://archive.apache.org/dist/skywalking/. The mirror is only providing latest version, it changes from time to time.

Reply:
Thanks for the information. Fixed